tuist/tuist
# Update resource accessor documentation to clarify that on-the-fly generation is not supported

Related to #2809 

### Short description 📝

Updates the documentation per [comment](https://github.com/tuist/tuist/issues/2809#issuecomment-1281693991) to make it clear on-the-fly resource accessor generation is not currently supported. This might be important for adopters to know.

### How to test the changes locally 🧐

This is a documentation tweak; you should be able to see it by running `docusaurus`. Here is a screenshot of the change:

<img width="996" alt="image" src="https://user-images.githubusercontent.com/78372052/201503844-9dc89c79-486e-4747-b4f5-a18adecccf89.png">

### Checklist ✅
